core/unit_test/tstNeighborList.hpp

```diff
@@ -903,15 +920,25 @@ TEST( TEST_CATEGORY, linked_cell_stencil_test ) { testLinkedCellStencil(); }
 //---------------------------------------------------------------------------//
 TEST( TEST_CATEGORY, verlet_list_full_test )
 {
-    testVerletListFull<Cabana::VerletLayoutCSR>();
-    testVerletListFull<Cabana::VerletLayout2D>();
+    testVerletListFull<Cabana::VerletLayoutCSR, Cabana::TeamOpTag>();
+    testVerletListFull<Cabana::VerletLayout2D, Cabana::TeamOpTag>();
+
+#ifndef KOKKOS_ENABLE_HIP
```

Comment:
Good - we will want this for now. add a `fixme` note so we can remove this when HIP is ready to go.

Reply:
Done